hardware_enablement/kmm-kernel-module-management.adoc

```diff
@@ -65,6 +65,9 @@ include::modules/kmm-building-and-signing-a-moduleloader-container-image.adoc[le
 .Additional resources
 For information on creating a service account, see xref:https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-creating-service-accounts.html#service-accounts-managing_understanding-service-accounts[Creating service accounts].
 
+// Added for TELCODOCS-1277
+include::modules/kmm-upgrading-kernel-modules-without-a-reboot.adoc[leveloffset=+1]
```

Comment:
Do we want to change the filename as well?

Reply:
Changed.